usaspending_api/data_load/generic_loaders.py

```diff
@@ -0,0 +1,67 @@
+from usaspending_api.data_load.data_load_helpers import setup_load_lists, setup_mass_load_lists
+
+
+def insert_recipient_locations(cursor, load_objects):
+    columns, values = setup_mass_load_lists(load_objects, "recipient_location")
+    recipient_location_sql = "INSERT INTO references_location {} VALUES {} RETURNING location_id;".format(
+        columns, values
+    )
+
+    cursor.execute(recipient_location_sql)
+    return [tup[0] for tup in cursor.fetchall()]
+
+
+def insert_recipients(cursor, load_objects):
+    columns, values = setup_mass_load_lists(load_objects, "legal_entity")
+    recipient_sql = "INSERT INTO legal_entity {} VALUES {} RETURNING legal_entity_id;".format(columns, values)
+
+    cursor.execute(recipient_sql)
+    return [tup[0] for tup in cursor.fetchall()]
+
+
+def insert_place_of_performance(cursor, load_objects):
+    columns, values = setup_mass_load_lists(load_objects, "place_of_performance_location")
+    recipient_sql = "INSERT INTO references_location {} VALUES {} RETURNING location_id;".format(columns, values)
+
+    cursor.execute(recipient_sql)
+    return [tup[0] for tup in cursor.fetchall()]
+
+
+def insert_award(cursor, load_object):
+    columns, values, pairs = setup_load_lists(load_object, "award")
+    generate_matching_award_sql = "INSERT INTO awards {} VALUES {} RETURNING id".format(columns, values)
+    cursor.execute(generate_matching_award_sql)
+    return cursor.fetchall()[0][0]
+
+
+def update_transaction_fpds(cursor, load_object):
+    columns, values, pairs = setup_load_lists(load_object, "transaction_fpds")
+    transaction_fpds_sql = "UPDATE transaction_fpds SET {} " "where detached_award_procurement_id = {}".format(
```

Comment:
Black reformat booger (to keep the AST identical it cannot combine the strings)
Need to watch for these:

```suggestion
    transaction_fpds_sql = "UPDATE transaction_fpds SET {} where detached_award_procurement_id = {}".format(
```